### ISSUE TITLE
First calibration doesn‘t work

### ISSUE THREAD

**Issue by winni67** [NightscoutFoundation/xDrip#345]
Hello, for me, the calibration of xDrip+ doesn’t work since the wizard for calibration was introduced. When xDrip+ is ready for calibration and I measure a blood glucose of 90 mg/dl then xDrip+ shows after calibration LOW. How can this happen? The old version from last year worked really good, but since a few month the first calibration in xDrip+ doesn’t work.

**Comment by tzachi-dar** [comment]
What sensor type?

**Comment by winni67** [comment]
Dexcom G5. I have to mention, that I set the new sensor some hours before, but without mounting the transmitter on it, I use a dummy instead. I mount the transmitter on the new sensor ten minutes before starting the new sensor in xDrip+.

**Comment by jamorham** [comment]
I'm pretty sure with the G5 and maybe also with the G4 you can't swap the transmitter to the sensor with such a short space of time you will get very messed up raw values because the G5 has internal electrical calibration. We've seen this before.

Try having the transmitter connected for at least an hour before attempting calibration, ideally the standard 2 hour warm up.

Also please let me know the version of xDrip you are currently running? Thanks.

**Comment by winni67** [comment]
I use
Version d8e7097-2018.01.22
Code: 1801222248

**Comment by winni67** [comment]
Today I tried it with a 2 hour warm up. I had the same issue, The glucose values after calibration (103 mg/dl) goes to low, but no active insulin or carbs were there to change the glucose value. After the next 15 minutes I went to a xDrip+ glucose value of 50 mg/dl. Thats not possible. A second calibration 65 minutes after the first calibration shows a glucose value of 89 mg/dl. The first hours, xDrip+ shows extreme low values. That's new for me, because I use xDrip+ for more than 16 month and this problem occoured with the xDrip version, that includes the startup wizzard for calibration. Perhaps I should use one of the older xDrip+ versions the next time to test if this problem disappear then.

**Comment by Deksa** [comment]
I think I've met the same problem, or related. I tried to put the data together in the picture. 

I have the same xDrip version as winni67: 
Version d8e7097-2018.01.22
Code: 1801222248

I'm thinking if.. when restarting the sensor in the Dex receiver the transmitter will destroy the calibration data and the sensor values from memory? This version of transmitter stores three (?) hours of data if not connecting. In my example from this morning the values started to make sense just a bit after I had restarted the sensor in Dex receiver.

This G5 transmitter is manufactured 2017-08-30.

<img width="1680" alt="nayttokuva 2018-3-21 kello 14 45 56" src="https://user-images.githubusercontent.com/22570897/37710537-9d5f0152-2d16-11e8-8d0e-7c69c427f823.png">
